docker/intel/Dockerfile

```diff
@@ -13,19 +13,21 @@ RUN apt-get update && apt-get install -y \
         git \
         ccache \
         libfftw3-dev \
+        python3 \
+        gfortran \
```

Comment:
Do we still need it ?

Reply:
No, we can revert this change. 
Clean up the docker `ENV` should be made in another PR

Reply:
Does this resolve #321 ?

Reply:
> Does this resolve #321 ?

I think yes, we can conclude that kokkos-fft is fine, the problem is on the Spack side.

Reply:
> Ok so we remove all changes in this Dockerfile ?

Yes, please